File: Examples/Ruby/Images/get_images.rb
require 'aspose_pdf_cloud'

class Image

  include AsposePDFCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @pdf_api = PdfApi.new  
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  # Read document images.
  def get_images
    file_name = "SampleImage.pdf"
    upload_file(file_name)

    page_number = 1
    response = @pdf_api.get_images(file_name, page_number)
  end

end

image_obj = Image.new()
puts image_obj.get_images

File: Examples/PHP/Images/GetImages.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\PDF\PdfApi;
use Aspose\PDF\AsposeApp;

class Images {

	public $pdfApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->pdfApi = new PdfApi();
	}

	public function getImages() {
		// Upload file to Aspose Cloud Storage
		$fileName = "SampleImage.pdf";
		Utils::uploadFile($fileName);
		
		$result = $this->pdfApi->GetImages($fileName, $pageNumber = 1, $storage = "", $folder = "");
		print_r ( $result );
	}
}

$images = new Images();
$images->getImages();

?>
//ExEnd:
